### DISCUSSION TITLE
Copilot did't do anything at all!

### DISCUSSION THREAD

**Discussion by tobylaifun** [community/community#10856] [Copilot Conversations]
<img width="730" alt="image" src="https://user-images.githubusercontent.com/60289741/151932883-81366588-de8a-4bc6-856f-807bb656ba75.png">

I logged in with my Github account, then I just let it print "hello" in python and it did not work.
So I use Javascript for testing, I pressed all the keyboard bindings that copilot had.
I checked the output of Copilot, but there was nothing...
I ask my friend, but he said copilot worked on his computer and he can use it by a `Enter`

**Comment by tobylaifun** [comment]
Every time I launch vscode, it look like this.
<img width="408" alt="image" src="https://user-images.githubusercontent.com/60289741/151933869-8095de91-0f81-482c-a5bf-d07dd5e6d56b.png">
I am in the GitHub Copilot technical preview.
It asks me to login everytime I launch.
<img width="332" alt="image" src="https://user-images.githubusercontent.com/60289741/151934010-272512a8-18c3-4639-b903-a0fde1203b3c.png">
Is it a vscode bug or a copilot bug?!
I could not login at all!

Github auth:
<img width="829" alt="image" src="https://user-images.githubusercontent.com/60289741/151934144-2129d1a4-4897-4e2c-ac82-2a37d6a03921.png">

**Comment by tobylaifun** [comment]
<img width="828" alt="image" src="https://user-images.githubusercontent.com/60289741/151934301-9cf10f1f-547a-4e3c-8c28-508f17fdf621.png">

**Comment by tobylaifun** [comment]
I can't even log into setting sync and live share.
Is it really a vscode bug?

**Comment by davecheney** [comment]
Can you please past the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by tobylaifun** [reply]
As I said,nothing there.
<img width="833" alt="image" src="https://user-images.githubusercontent.com/60289741/151950207-2ac838c7-8c2b-46bf-83be-7b2b9d78096b.png">

**Reply by davecheney** [reply]
Thanks for confirming. I suspect something is blocking your connection to sign in to vscode. If that is blocked then the activation of the copilot extensions will fail.

**Reply by tobylaifun** [reply]
I launched [devsidecar](https://github.com/docmirror/dev-sidecar), a app to improve the speed of github.
But if I don't, I can't connect to it.
I don't think devsidecar is the reason
